File: 100-Days-Series/Day 37/RemoveSpace.java
public class RemoveSpace {
    // Function for remove spaces from a string
    public static String getRemoveSpaces(String str){
        String temp="";
        for(int i=0;i<str.length();i++){
            char ch=str.charAt(i);
            if(ch!=' ')
                temp+=ch;
        }
        return temp;
    }
    // Main Method
    public static void main(String [] args){
        // Let's Start here
        String str;
        System.out.print("Enter a String : ");
        java.util.Scanner sc=new java.util.Scanner(System.in);
        str=sc.nextLine();
        System.out.println("String with spaces : "+str);
        System.out.println("String withour spaces : "+getRemoveSpaces(str));
    }
}

File: 100-Days-Series/Day 37/RemoveSpace.py



# Function to remove spaces from a string
def getRemoveSpaces(str):
    temp=""
    for i in range(0,len(str),1):
        if str[i]!=' ':
            temp+=str[i]
    return temp

# Let's Start here
str=input("Enter a string : ")
print("String with spaces : ",str)
print("String withour spaces : ",getRemoveSpaces(str))